packages/ckeditor5-utils/src/dom/position.ts

```diff
@@ -115,7 +115,7 @@ import { isFunction } from 'lodash-es';
  */
 export function getOptimalPosition( {
 	element, target, positions, limiter, fitInViewport, viewportOffsetConfig
-}: Options ): Position | null {
+}: Options ): DomPoint | null {
```

Comment:
I'm not sure if this would not be confusing if we keep the old file and module name. WDYT? OTOH this would be a breaking change, but is this already a breaking change?

Reply:
I think that keeping the file with old name is not confusing - changing only the interface name will make this breaking change just minor, not a major one.